File: CodingPractice-master/leetcode/algorithms/python/0709_ToLowerCase.py
# -*- coding: utf-8 -*-
# @Author: Yan_Daojiang
# @Date:   2019-05-08 18:56:18
# @Last Modified by:   Yan_Daojiang
# @Last Modified time: 2019-05-09 11:41:44
class Solution:
    def toLowerCase(self, str: str) -> str:
        strlist=list(str)
        for i in range(len(strlist)):
            if strlist[i]>='A' and strlist[i]<='Z':
                strlist[i]=chr(ord(strlist[i])+32)
        return "".join(strlist)

File: CodingPractice-master/leetcode/algorithms/cpp/0709_ToLowerCase.cpp
/*
* @Author: Yan_Daojiang
* @Date:   2019-05-08 18:58:12
* @Last Modified by:   Yan_Daojiang
* @Last Modified time: 2019-05-08 18:58:31
*/
class Solution {
public:
    string toLowerCase(string str) {
        for(int i=0;i<str.length();i++)
            if(str[i]>=65&&str[i]<=90)
                str[i]=str[i]+32;
        return str;
    }
};